lib/order.rb

```diff
@@ -0,0 +1,66 @@
+require 'awesome_print'
+require 'csv'
+require_relative 'customer'
+
+class Order
+  attr_reader :id, :customer, :products
+  attr_accessor :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id #integer
+    @products = products #hash
+    @customer = customer #instance of customer
+    status_options = [:pending, :paid, :processing, :shipped, :complete]
+    unless status_options.include? fulfillment_status
```

Comment:
Consider using a postfix conditional and putting this as the first item in your constructor as a Guard clause.